querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java

```diff
@@ -59,9 +61,14 @@
      * @param keywords keywords to be used
      */
     @Inject
-    public EntitySerializer(TypeMappings mappings, @Named("keywords") Collection<String> keywords) {
+    public EntitySerializer(TypeMappings mappings, @Named("keywords") Collection<String> keywords, @Named(CodegenModule.CASE_TRANSFORMER_CLASS) String caseTransfomerClass) {
         this.typeMappings = mappings;
         this.keywords = keywords;
+        try {
+           this.caseTransformer = (CaseTransformer) Class.forName(caseTransfomerClass).newInstance();
+        } catch (Exception e) {
```

Comment:
What do you think about this broad catch clause?
Do you maybe think it can be split up?
I think at least a catch clause for RuntimeException (and rethrowing it) would be better.